src/ui/dialog-vkbasalt.ui

```diff
@@ -0,0 +1,820 @@
+                                    <property name="upper">100</property>
+                                  </object>
+                                </property>
+                              </object>
+                            </child>
+                          </object>
+                        </child>
+                      </object>
+                    </child>
+                  </object>
+                </child>
+                <child>
+                  <object class="AdwExpanderRow" id="clut">
+                    <property name="show_enable_switch">True</property>
+                    <property name="title" translatable="yes">Color LookUp Table</property>
+                    <child>
+                      <object class="AdwPreferencesRow">
+                        <child>
+                          <object class="AdwActionRow">
+                            <property name="title" translatable="yes">Path</property>
+                            <child>
+                              <object class="GtkEntry" id="input_entry">
+                                <property name="valign">center</property>
+                                <property name="editable">False</property>
+                              </object>
+                            </child>
+                            <child>
+                              <object class="GtkMenuButton">
+                                <property name='valign'>center</property>
+                                <property name="tooltip-text" translatable="yes">Select</property>
+                                <property name="icon-name">info-symbolic</property>
+                                <property name="popover">popover_clut</property>
+                                <style>
+                                  <class name="flat"/>
+                                </style>
+                              </object>
+                            </child>
+                            <child>
+                              <object class="GtkButton" id="lut_file_path">
+                                <property name='valign'>center</property>
+                                <property name="tooltip-text" translatable="yes">Select</property>
+                                <property name="icon-name">paper-symbolic</property>
+                                <style>
+                                  <class name="flat"/>
+                                </style>
+                              </object>
+                            </child>
+                          </object>
+                        </child>
+                      </object>
+                    </child>
+                  </object>
+                </child>
+              </object>
+            </child>
+            <child>
+              <object class="AdwPreferencesGroup">
+                <property name="title" translatable="yes">Miscellaneous</property>
```

Comment:
I don't like having a separate group for one item. I'd just remove the group's title, so that there's still some separation but it isn't as jarring.